src/lib/statistics/statistics_objects/counting_quotient_filter.cpp

```diff
@@ -4,8 +4,6 @@
 #include <iostream>
 #include <string>
 
-#include "boost/variant.hpp"

```

Comment:
boost variant is broken in 1.70 and fixed again in 1.71, which is not even available in the ppa.